pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -0,0 +1,128 @@
+	metav1.ListMeta `json:"metadata,omitempty"`
+	// List of ScrapeConfigs
+	Items []*ScrapeConfig `json:"items"`
+}
+
+// DeepCopyObject implements the runtime.Object interface.
+func (l *ScrapeConfigList) DeepCopyObject() runtime.Object {
+	return l.DeepCopy()
+}
+
+// ScrapeConfigSpec is a specification of the desired configuration for a scrape configuration.
+// +k8s:openapi-gen=true
+type ScrapeConfigSpec struct {
+	// StaticConfigs list of labeled statically configured targets for this job.
+	StaticConfigs []StaticConfig `json:"staticConfigs,omitempty"`
+	// FileSDConfigs list of file service discovery configurations.
+	FileSDConfigs []FileSDConfig `json:"fileSDConfigs,omitempty"`
+	// HTTPSDConfigs list of HTTP service discovery configurations.
+	HTTPSDConfigs []HTTPSDConfig `json:"httpSDConfigs,omitempty"`
+	// RelabelConfigs to apply to samples before scraping.
+	// Prometheus Operator automatically adds relabelings for a few standard Kubernetes fields.
+	// The original scrape job's name is available via the `__tmp_prometheus_job_name` label.
+	// More info: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
+	RelabelConfigs []*v1.RelabelConfig `json:"relabelings,omitempty"`
+	// MetricsPath HTTP path to scrape for metrics. If empty, Prometheus uses the default value (e.g. /metrics).
+	MetricsPath string `json:"metricsPath,omitempty"`
+	// HonorTimestamps controls whether Prometheus respects the timestamps present in scraped data.
+	HonorTimestamps *bool `json:"honorTimestamps,omitempty"`
+	// HonorLabels chooses the metric's labels on collisions with target labels.
+	HonorLabels *bool `json:"honorLabels,omitempty"`
+}
+
+// StaticConfig defines a prometheus static configuration
+// See https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config
+// +k8s:openapi-gen=true
+type StaticConfig struct {
+	// List of targets for this static configuration
+	Targets []Target `json:"targets"`
+	// Labels assigned to all metrics scraped from the targets.
+	// +optional
+	Labels map[v1.LabelName]string `json:"labels,omitempty"`
+}
+
+// FileSDConfig defines a prometheus file service discovery configuration
+// See https://prometheus.io/docs/prometheus/latest/configuration/configuration/#file_sd_config
+// +k8s:openapi-gen=true
+type FileSDConfig struct {
+	// List of files to be used for file discovery. Recommendation: use absolute paths. While relative paths work, the
+	// prometheus-operator project can't guarantee that the working directory will stay the same over time.
+	// Files must be mounted using Prometheus.ConfigMaps or Prometheus.Secrets.
+	// +kubebuilder:validation:MinItems:=1
+	Files []string `json:"files"`
+	// RefreshInterval configures the refresh interval at which Prometheus will reload the content of the files.
+	// +optional
+	RefreshInterval v1.Duration `json:"refreshInterval,omitempty"`
```

Comment:
```suggestion
	RefreshInterval *v1.Duration `json:"refreshInterval,omitempty"`
```

Reply:
Thanks! Fixed in 576480855c09a6423659447eb6b9da15261a753a